CODEOWNERS

```
@@ -16,17 +16,20 @@ Cargo.toml @ryanolson @grahamking @paulhendricks @biswapanda @tmonty12 @guanluo
 # Dynamo deploy
 /deploy/ @hutm @biswapanda @ishandhanani @julienmancuso @hhzhang16 @nnshah1 @mohammedabdulwahhab
 
-# All examples
-/examples/ @hutm @biswapanda @ishandhanani @julienmancuso @hhzhang16 @nnshah1 @tanmayv25 @piotrm-nvidia @ptarasiewiczNV @ryanolson @grahamking @paulhendricks @biswapanda @tmonty12 @guanluo @rmccorm4 @tedzhouhk @alec-flowers @kkranen @sshchoi @PeaBrane @mohammedabdulwahhab @ai-dynamo/Devops
-
-# TRTLLM
-/examples/tensorrt_llm/ @tanmayv25 @rmccorm4 @guanluo @richardhuo-nv
-
-# Multimodal
-/examples/multimodal/ @indrajit96 @whoisj @krishung5
+# Examples
+/examples/ @ai-dynamo/Devops @nnshah1 @whoisj
```

self-requesting to be on top level here, helps me stay on top of all the moving changes even if I can't review every single one
```suggestion
/examples/ @ai-dynamo/Devops @nnshah1 @whoisj @rmccorm4 
```